ddtrace/contrib/internal/google_genai/patch.py

```diff
@@ -33,9 +33,15 @@ def traced_generate(genai, pin, func, instance, args, kwargs):
         "%s.%s" % (instance.__class__.__name__, func.__name__),
         provider=provider_name,
         model=model_name,
-        submit_to_llmobs=False,
-    ):
-        return func(*args, **kwargs)
+        submit_to_llmobs=True,
+    ) as span:
+        resp = None
+        try:
+            resp = func(*args, **kwargs)
+            return resp
+        finally:
+            if integration.is_pc_sampled_llmobs(span):
```

Comment:
https://github.com/DataDog/dd-trace-py/pull/13851 removed any need to do the llmobs sampling check here!